test/parallel/test-http-set-timeout-server.js

```diff
@@ -1,37 +1,35 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var http = require('http');
-var net = require('net');
 
-var tests = [];
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const net = require('net');
+
+const tests = [];
 
 function test(fn) {
-  if (!tests.length)
+  if (!tests.length) {
     process.nextTick(run);
+  }
   tests.push(fn);
 }
 
 function run() {
-  var fn = tests.shift();
-  if (fn) {
-    console.log('# %s', fn.name);
-    fn(run);
-  } else {
-    console.log('ok');
-  }
+  const fn = tests.shift();
+  if (fn) fn(run);
+  else process.exit(0);
```

Comment:
This mean that the assertions in `process.on('exit'...)` never fire. I don't think you can do the test this way.